tests/integration-service/integration.go

```diff
@@ -153,27 +153,26 @@ var _ = framework.IntegrationServiceSuiteDescribe("Integration Service E2E tests
 		})
 
 		When("An snapshot of push event is created", func() {
+			// disable this case to not run GCL test and then enable after https://github.com/konflux-ci/integration-service/pull/733 is merged
```

Comment:
Instead of doing this copy paste, we can just mark test as skipped https://onsi.github.io/ginkgo/#skipping-specs

Reply:
Thanks. Updated. Let's see if it can be skipped.

Reply:
Yes, it is skipped.